source/gcx_apps/meeting_room_monitor_app/app.c

```diff
@@ -94,6 +97,21 @@ static uint32_t send_data_task(void)
     return period_ms;
 }
 
+
+/**
+ * @brief   Task resets the LED
+ * @return  next period
+ */
+static uint32_t reset_led_task(void)
```

Comment:
Maybe we could find a better name for this function.  A name that indicates switching off the led. 
This is optional and up to you.

Reply:
changed the name